brd-android/app/src/main/java/com/breadwallet/ui/home/HomeScreen.kt

```diff
@@ -182,12 +187,21 @@ object HomeScreen {
         }
 
         object GoToSupportForm : F(), NavigationEffect {
-            override val navigationTarget = NavigationTarget.AlertDialog(
-                titleResId = R.string.SupportForm_helpUsImprove,
-                messageResId = R.string.SupportForm_feedbackAppreciated,
-                positiveButtonResId = R.string.Button_submit,
-                negativeButtonResId = R.string.SupportForm_notNow,
-                textInputPlaceholderResId = R.string.SupportForm_pleaseDescribe
+            override val navigationTarget = NavigationTarget.FabriikGenericDialog(
+                FabriikGenericDialogArgs(
+                    titleRes = R.string.SupportForm_helpUsImprove,
+                    descriptionRes = R.string.SupportForm_feedbackAppreciated,
+                    textInputHintRes = R.string.SupportForm_pleaseDescribe,
```

Comment:
I think we do not need an input view inside the dialog. This effect could be changed and we can show feedback in similar way as we do on the Menu screen. Here we can just replace positive button text with Continue and open Email client when user click on it